modules/apps/frontend-js/frontend-js-tooltip-support-web/src/main/resources/META-INF/resources/index.tsx

```diff
@@ -114,23 +117,15 @@ const TooltipProvider = () => {
 
 	const saveTitle = (element: HTMLElement) => {
 		if (element) {
-			const title = element.getAttribute('title');
+			const title =
+				element.getAttribute('title') ||
+				element.getAttribute('data-title');
 
 			if (title) {
 				element.setAttribute('data-restore-title', title);
-				element.removeAttribute('title');
-			}
-			else if (element.tagName === 'svg') {
```

Comment:
Is it safe to remove this special handling of `svg`s?

Reply:
I actually wasn't quite sure why we had this custom SVG handling since I think we are still adding the title attr to the svg element. I removed this for simplicity for now, I want to investigate further before merging this change.

@julien it looks like you were the last person to touch this code, any chance you remember why we had this particular case?